rllib/BUILD

```diff
@@ -1228,91 +1228,197 @@ py_test(
 
 
 py_test(
-    name = "examples/autoregressive_action_dist", main = "examples/autoregressive_action_dist.py",
+    name = "examples/autoregressive_action_dist_tf",
```

Comment:
Most example scripts (where applicable) now use the `--as-test` flag to indicate that learning targets (`--stop-reward`) should be achieved and checked when running this example.
Also, most example scripts (where applicable) are tested for both tf and torch (using the `--torch` flag).